script/UpdateCBridgeFacetPackedToken.s.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: UNLICENSED
```

Comment:
I just noticed this. We usually have MIT License on all files. Here we have UNLICENSED. Why? And should it be coherent?

Reply:
Many of the Update scripts contain this License Identifier, if we want to change that we should do it in one PR on all files.